openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/directives/arrow-tab.directive.spec.ts

```diff
@@ -133,12 +133,14 @@ describe('ArrowTabDirective', () => {
 
         it('should not error trying to select next', () => {
             KeybindingTestUtils.pressKey('ArrowDown');
-            expect(document.activeElement === document.body || document.querySelector('mat-dialog-container')).toBeTrue();
+            expect(document.activeElement === document.body
+                || document.activeElement ===document.querySelector('mat-dialog-container')).toBeTrue();
```

Comment:
The lack of SPACE! GAHHH!

```suggestion
            expect(document.activeElement === document.body
                || document.activeElement === document.querySelector('mat-dialog-container')).toBeTrue();
```

Reply:
I PUSHED A DUMB FIX BEFORE I SAW YOUR COMMENT!